src/GraphQL/Introspection/__DirectiveArgument.cs

```diff
@@ -44,8 +43,7 @@ public __DirectiveArgument()
                     var argumentDefinition = directiveDefinition!.Arguments!.Find(argument.Name);
 
                     var ast = argumentDefinition!.ResolvedType!.ToAST(argument.Value);
-                    string result = AstPrinter.Print(ast);
-                    return string.IsNullOrWhiteSpace(result) ? null : result;
+                    return ast.Print();
```

Comment:
This makes sense, but `Print` could be internal for this purpose.  The extension method is just supporting introspection code.

Reply:
1. Introspection usage by `__DirectiveArgument` and `__InputValue` to return GraphQL-formatted string representing the default value
2. `SchemaPrinter` that eventually will be completely rewritten
3. `OverlappingFieldsCanBeMerged` uses it to compare arguments (dirty implementation, should be rewritten)
4. `ExecutionHelper` uses it to buil error message: `throw new ArgumentOutOfRangeException(nameof(input), $"Expected object value for '{inputObjectGraphType.Name}', found not an object '{input.Print()}'.");`

Reply:
`Print` is internal now.